views/templates/show.html

```diff
@@ -0,0 +1,53 @@
+<!-- show.html -->
+<div class="container">
+
+  <!-- Add Books Form -->
+  <div ng-show="editing" class="row container">
+    <form class="form-horizontal centered" ng-submit="bsc.updateBook(bsc.book)">
+
+        <input class="btn btn-success btn-lg" type="submit" name="submit" value="Save">
+          <label class="control-label"> Title
+            <input class="form-control input-lg" type="text" ng-model="bsc.book.title" name="title" value="" placeholder="book title">
```

Comment:
It's best practice not to use a placeholder to say what should be put there, but instead to give an example... "Macbeth" or something.